gatsby/should-update-scroll.js

```diff
@@ -0,0 +1,16 @@
+const shouldUpdateScroll = ({
+  prevRouterProps,
+  routerProps: { location },
+  getSavedScrollPosition,
+}) => {
+  if (
+    location.pathname.includes('/install/') &&
```

Comment:
would using `startsWith` make more sense here to avoid disabling scroll on any page url paths that contain `/install/` ?

Reply:
oooo sorry just saw this one, i think it would actually make sense to do that and can put that in a fix